www/caddy/src/opnsense/mvc/app/views/OPNsense/Caddy/reverse_proxy.volt

```diff
@@ -249,7 +249,7 @@
         });
 
         $("#handle\\.HttpTls").change(function() {
-            if ($(this).val() === "0") {
```

Comment:
First it checked if it is http (0)

But since http (0) and h2c (2) are both unencrypted protocols, they should not show the TLS options.

So now I check for !https (1) instead.

They're numbers because it has been checkboxes before that I turned into selectpickers without a migration by assigning them values. I never had h2c on the radar so now we are here.

Reply:
Sure I can add comments there. <3